addons/web/static/src/core/code_editor/code_editor.js

```diff
@@ -42,6 +44,7 @@ export class CodeEditor extends Component {
         },
         maxLines: { type: Number, optional: true },
         sessionId: { type: [Number, String], optional: true },
+        record: { type: Object }
```

Comment:
add a coma at the end of the line so the next person who will add prop will not modify your line :+1: